fidesctl/src/fidesctl/cli/core_commands.py

```diff
@@ -213,8 +221,17 @@ def scan(
     default=False,
     help="Strictly validate annotation inputs.",
 )
+@click.option(
```

Comment:
since this is being used twice, can you move it to the `options.py` file and import it for use here?

Reply:
good call, thanks !